app/controllers/api/v2/users_controller.rb

```diff
@@ -18,7 +18,9 @@ def update
       end
 
       def accept_transition_screen
-        @user.accept_transition_screen!
+        if @user.state == 'new'
+          @user.accept_transition_screen!
+        end
       end
 
       def accept_rollover_screen
```

Comment:
We should do the same for the rollover screen.

Reply:
Cool i'll add that in and edit the card.